Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM centos:6
```

Comment:
I'm afraid the "folder" approximation has a major flaw:
```
Step 2 : COPY .. /opt/iota-thinkingthings
Forbidden path outside the build context: .. (/var/lib/docker/tmp)
```
You can't use COPY commands from an inner folder, so if you want to copy the entire repository, the Docker file is bound to be at the repository's root.

I'm reverting the changes, and NTC.